content/common/cli-reference.md

```diff
@@ -1068,7 +1100,51 @@ TRIGGER OPTIONS:
 
 {{ blockEnd }}
 
-> **Please note:** If the `-f` or `--from` options do not accurately infer the intended registry or `.toml` file for your application, then you can explicitly specify either the `--from-registry` or  `--from-file` options to clarify this.
+> **Please note:** If the `-f` or `--from` options do not accurately infer the intended registry or `.toml` file for your application, then you can explicitly specify either the `--from-registry` or `--from-file` options to clarify this.
+
+### Watch
+
+In order to configure the files that are watched you need to configure the `component.build.watch` parameter in your spin.toml.
+
+```toml
+[component.build]
+# Example watch configuration for a Rust application
+watch = ["src/**/*.rs", "spin.toml", "Cargo.toml"]
+```
+
+{{ tabs "spin-version" }}
+
+{{ startTab "v1.1.0"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin watch --help
+
+spin-watch
+Rebuild and restart the Spin application when files changes
```

Comment:
Suggested text: "Build and run the Spin application, rebuilding and restarting it when files change"

Why?  Because the current text doesn't actually make it clear that this _runs the app_ - it sounds like something you do to an app that is _already running_.

Sorry for not catching it on the original PR - I somehow missed that coming out of draft, so didn't wordsmith at the time.

Reply:
Good catch. I'll update the CLI to the new text.